### ISSUE TITLE
How to exploit output in Forge viewer?

### ISSUE THREAD

**Issue by leefsmp** [jeremytammik/RoomVolumeDirectShape#1]
Thanks for a useful sample! This overlooks a quite important aspect of the workflow: how to exploit the output file to visualize rooms in the Forge viewer...?

**Comment by jeremytammik** [comment]
Yes, absolutely. This is in collaboration with Michael Beale, and I am leaving all DA4R related aspects of this up to him. Maybe he has an answer for us both? @micbeale on twitter, can't find him on github.

**Comment by jeremytammik** [comment]
i asked him to respond to us here...

**Comment by wallabyway** [comment]
Ah - that would be me (@micbeale) ;-)

I need to update this repo: https://github.com/wallabyway/rooms-spaces-revit-plugin
with Jeremy's directShape code and then blog about it.
My laptop is in for repairs (so no windows) but will tackle it once it's back !
;-)

**Comment by leefsmp** [comment]
The part that matters most to me is loading the gltf output of that code to the Forge viewer, I don't see that in your repo either? Did I miss smthg? btw - no spare machine at adsk!?! ;)

**Comment by leefsmp** [comment]
@micbeale: Hi wondering if you could provide an approximative ETA for that...? Thx
